src/components/HomePage/Banner.js

```diff
@@ -26,9 +26,9 @@ const useStyles = makeStyles(theme => ({
 		height: 'auto',
 		[theme.breakpoints.up('md')]: {
 			backgroundImage: `url(${hothBanner})`,
-			backgroundSize: '545px 329px',
+			backgroundSize: '1200px 500px',
```

Comment:
It seems like this is why the banner is unresponsive for small screens. We should probably maintain the split between the left portion with the text/info and the right section with the graphic and countdown

Reply:
In general, the SVG we export should only contain the parts that we cannot reproduce ourselves in HTML. That means we don't need to export the (large) background rectangle either.